server/main.lua

```diff
@@ -39,12 +39,35 @@ if Config.UseWolfknightRadar == true then
 	AddEventHandler("wk:onPlateScanned", function(cam, plate, index)
 		local src = source
 		local Player = QBCore.Functions.GetPlayer(src)
-		local bolo = GetBoloStatus(plate)
+		local PlayerData = GetPlayerData(src)
+		local vehicleowner = GetVehicleOwner(plate)
+		local bolo, title, boloid = GetBoloStatus(plate, title, boloid)
+		local warrant, owner, incidentid = GetWarrantStatus(plate, owner, incidentid)
+		local driverslicense = PlayerData.metadata['licences'].driver
+		local driverunlicensed = nil
+
 		if bolo == true then
-			TriggerClientEvent("wk:togglePlateLock", src, cam, true, bolo)
+			TriggerClientEvent('QBCore:Notify', src, 'BOLO ID: '..boloid..' | Title: '..title..' | Registered Owner: '..vehicleowner..' | Plate: '..plate, 'error', Config.WolfknightNotifyTime)
+		end
+		if warrant == true then
+			TriggerClientEvent('QBCore:Notify', src, 'WANTED - INCIDENT ID: '..incidentid..' | Registered Owner: '..owner..' | Plate: '..plate, 'error', Config.WolfknightNotifyTime)
+		end
+
+		if driverslicense == false then
+			driverunlicensed = true
+			if driverunlicensed == true then
+			TriggerClientEvent('QBCore:Notify', src, 'NO DRIVERS LICENCE | Registered Owner: '..vehicleowner..' | Plate: '..plate, 'error', Config.WolfknightNotifyTime)
+			end
 		end
+
+
+		if bolo or warrant or driverunlicensed == true then

```

Comment:
`driverunlicensed` variable is a bit redundant, you could simplify this by simply checking `not driverslicense` and then you can remove the driverunlicensed variable
```suggestion
		if bolo or warrant or not driverslicense then

```